### DISCUSSION TITLE
[Feature Request] Relay HTTP Codes from JSON URL

### DISCUSSION THREAD

**Discussion by ckocyigit** [grafana/grafana-infinity-datasource#93] [Ideas]
**Is your feature request related to a problem? Please describe.**
When working with json endpoints like the actuators from springboot the status of the service is also represented by the HTTP Code of the actuators url.

For example if service A is down

example.tld/backend/api/actuator/health/serviceA

will not work with the plugin since even though the result is not empty springboot will reply with HTTP Code 503 representing the status of the service.

**Describe the solution you'd like**
Is there a possibility to relay the http code to work within the panel instead of failing?

**Comment by yesoreyeram** [comment]
Thanks for this. Definitely seems a good idea to have this feature in an upcoming release.

**Reply by yesoreyeram** [reply]
Fixed in https://github.com/grafana/grafana-infinity-datasource/pull/1076
